### PR TITLE
HexEditor: Minor tweaks

### PR DESCRIPTION
**Pull request by AtkinsSJ**
I meant to start working on adding annotations, but got a bit distracted, so here are some assorted tweaks that won't belong in that PR, whenever it happens.

### CODE REVIEW

File: Userland/Applications/HexEditor/HexEditor.cpp [hunk 1]
```diff
@@ -724,15 +732,16 @@ void HexEditor::paint_event(GUI::PaintEvent& event)
             };
 
             painter.fill_rect(text_background_rect, background_color_text);
-            auto character = String::formatted("{:c}", isprint(cell_value) ? cell_value : '.').release_value_but_fixme_should_propagate_errors();
 
             if (m_edit_mode == EditMode::Text)
                 draw_cursor_rect();
 
+            auto character = is_ascii_printable(cell.value) ? cell.value : '.';
+            auto character_sv = StringView { reinterpret_cast<char const*>(&character), 1 };
```
Comment (LucasChollet) [inline Userland/Applications/HexEditor/HexEditor.cpp:740]: Can't you just make `character` be a `char const` instead of casting?

Comment (AtkinsSJ) [reply Userland/Applications/HexEditor/HexEditor.cpp]: Yup! That's better. :^)

**Review by LucasChollet — APPROVED** [review]
LG!
(The last commit is :yak_kiss: btw)

(PS: I won't tell anyone, but you forgot to fixup the change :shushing_face:)

**Comment by AtkinsSJ** [comment]
🤦‍♂️ I don't know how I did that. I'll sort that out in a bit!

**Comment by AtkinsSJ** [comment]
I rebased on master accidentally, but otherwise the only change was to combine those two commits that should have been one commit in the first place. :sweat_smile: So, should be good after CI.

**Pull request merged**